cpu/sam0_common/include/sam_usb.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2018 Freie Universität Berlin
+ * Copyright (C) 2018 Inria
```

Comment:
You should probably update these copyrights.

Reply:
Fixed, thanks